week01/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("Hello, World!")
+
+	varString := "GO"
```

Comment:
In Go, variable names tend to be very short, and they don't usually include the name of the type and that it is a var. For example, `varString`, would be accepted in Go organization, as it is not idiomatic. It **is** idiomatic, to use short, single character, variable names that represent the type in question. For example, `w` is often used as a variable name for `io.Writer` or `r` being used for `io.Reader`.

```suggestion
	s := "GO"
```